heclib/javaHeclib/src/Hec_zopen.c

```diff
@@ -35,13 +35,6 @@ JNIEXPORT int JNICALL Java_hec_heclib_util_Heclib_Hec_1zopen
         {
             *status = 7;
         }
-        else {
-            // solaris only.. default to DSS6
-#ifdef __sun__
-            * status = 6;
```

Comment:
I've added a note here to document this change.

https://www.hec.usace.army.mil/confluence/dssdocs/dsscprogrammer/release-notes